src/DetailsView/reports/components/report-sections/details-section.tsx

```diff
@@ -12,6 +12,12 @@ export type DetailsSectionProps = Pick<SectionProps, 'pageUrl' | 'description' |
 
 export const DetailsSection = NamedSFC<DetailsSectionProps>('DetailsSection', props => {
     const { pageUrl, description, scanDate, toUtcString } = props;
+    const scanDateUTC: string = toUtcString(scanDate);
+    const screenReaderTexts = {
+        targetPageLink: `Target Page: ${pageUrl}`,
```

Comment:
I think you should drop the ':' here. NVDA doesn't read them so it's fine but JAWS **DO** read them which sounds weird: "target page colon http://...."

Reply:
Thats what @ferBonnin told me to add